test/cypress/integration/watermarkModal.spec.js

```diff
@@ -0,0 +1,331 @@
+const ID = {
```

Comment:
Let's skip these tests until 6.0 is out so that we don't need to commit the `lib` folder to git.

Can you remove the lib folder and add it back to the .gitignore please?

Reply:
right now. tests can be ran on pipeline but they can't be ran locally, and i'm having trouble setting up docker locally. sure. i am fine with this.

Reply:
should i comment out the end to end test phase in the pipeline?